src/Expensify.tsx

```diff
@@ -130,6 +131,7 @@ function Expensify({
 
     const onSplashHide = useCallback(() => {
         setIsSplashHidden(true);
+        Performance.markEnd(CONST.TIMING.SIDEBAR_LOADED);
```

Comment:
On splash hide we are considering the sidebar loaded(ie. performance entry name SIDEBAR_LOADED), wondering if that is correct or name should be different? Asking bcoz earlier we did it in `setSidebarLoaded`.

Reply:
@Pujan92 from one side - you are right, it's confusing that we use `SIDEBAR_LOADED` when splash screen gets hidden. But the start mark is added in `BaseSidebar` component (on mount).

So in my opinion it's better not to rename it 🙂